cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/default-service/AwsDefaults.service.tsx

```diff
@@ -0,0 +1,15 @@
+// React imports
+import React from 'react';
+import { AwsService, AWSVirtualMachine } from '../../../../../swagger-api';
+
+export class AwsDefaults extends React.Component {
+    // Find what the default osImage is
+    static selectDefalutOsImage = (osImages: AWSVirtualMachine[]) => {
+        return osImages[0];
```

Comment:
I think the osImages array could be empty, so maybe: `osImages && osImages.length > 0 ? osImages[0] : undefined`

Reply:
Right. I do return a 'undefined' at first. But as I used a `if (osImages.length !== 0) {` in AwsOrchestrator, I removed this undefined return.
Gonna use `static selectDefalutOsImage = (osImages: AWSVirtualMachine[]) => {
        return osImages && osImages.length > 0 ? osImages[0] : undefined;
    };`